lib/ansible/modules/cloud/amazon/ec2_lc.py

```diff
@@ -67,6 +67,10 @@
     description:
       - opaque blob of data which is made available to the ec2 instance
     required: false
+  user_data_path:
+    description:
+      - Path to the file that contains userdata for the ec2 instances
```

Comment:
`Path to the file that contains userdata for the ec2 instances. Multualive exclusive with I(user_data).`

And similar on the user_data `description:` block

The `I(...)` means italics, so it's clearer we are referring to a parameter.